plugin/src/main/java/com/yelp/codegen/KotlinGenerator.kt

```diff
@@ -176,7 +182,8 @@ class KotlinGenerator : SharedCodegen() {
         return codegenModel
     }
 
-    private fun addRequiredImports(codegenModel: CodegenModel) {
+    @VisibleForTesting
+    internal fun addRequiredImports(codegenModel: CodegenModel) {
```

Comment:
According to the [doc](https://developer.android.com/reference/android/support/annotation/VisibleForTesting) `@VisibleForTesting` without further specifications makes the decorated method private.

How is `internal` going to be used here?

Reply:
> According to the doc @VisibleForTesting without further specifications makes the decorated 
> method private.

Not really.
The doc says that you can optionally specify which would have been the desired visibility. If you don't specify the desired visibility, it will default to private:

> If not specified, the intended visibility is assumed to be private.

Anyway this desired visibility has no real effect on the bytecode and is mostly used by tools (e.g. the IDE) to show warnings on usages of those fields.